doc/sphinx/source/framework-integration-guides.rst

```diff
@@ -17,19 +17,19 @@ MXNet\*
 Compile MXNet with nGraph
 --------------------------
 
-.. important:: These instructions pick up from where the :doc:`install`
-   installation instructions left off, so they presume that your system already
-   has the library installed at ``$HOME/ngraph_dist`` as the default location.
-   If the |nGl| code has not yet been installed to your system, please go back
-   and return here to finish compiling MXNet with ``libngraph``.
+.. important:: As of version |version|, these instructions presume that your 
+   system already has the Library installed to the default location, as outlined 
```

Comment:
I think we should change the install docs to have the user set `-DCMAKE_INSTALL_PREFIX=~/ngraph_dist` as the default location. The cmake default install location is to install ngraph into the system folders as defined by each OS.

Reply:
fixed, we now specify manual compilation